ghstack/github_fake.py

```diff
@@ -347,12 +368,33 @@ def _set_default_branch(self, owner: str, name: str,
         repo = state.repository(owner, name)
         repo.defaultBranchRef = repo._make_ref(state, input['default_branch'])
 
+    def _add_labels(self, owner: str, name: str, number: GitHubNumber,
+                    input: AddLabelsInput) -> AddLabelsPayload:
+        state = self.state
+        repo = state.repository(owner, name)
+        pr = state.pull_request(repo, number)
+        pr.labels += [{'name': label} for label in input['labels']]
```

Comment:
Does GitHub deduplicate the labels if you relabel something with something it already has?

Reply:
Good question 🤔 I can test it tomorrow

Reply:
I'm guessing the answer is yes; if so then I'll update this mock to store it as a set instead of a list